src/api/cf.nim

```diff
@@ -26,12 +27,20 @@ const
   ## base url of the forgesvc endpoint
   modsBaseUrl = "https://addons-ecs.forgesvc.net/api/v2"
 
+
+converter getDependencies(json: JsonNode): int =
+  var id: int = json{"id"}.getInt()
```

Comment:
Also, do you have an example where the json returns an `id` property? According to my testing the `dependencies` property in my json is always a list of `addonId` and `type`.

Reply:
Old code. If you go straight to an addon (i.e. jei: https://addons-ecs.forgesvc.net/api/v2/addon/240630) you can see it lists the id, fileid, type, and addonId under it's dependencies. But if you just look at the file it only lists the addonId. The original hope was to pull the fileId and Id to bypass the second call to fetchModFiles, but it doesn't end up working out. Sorry for not catching that.